src/components/form/form.style.js

```diff
@@ -16,14 +19,49 @@ export const StyledAdditionalFormAction = styled.div`
       display: inline-block;
     `}
 
-    ${StyledButton} {
+    ${applyButtonLeftMargin}
+
+    && ${StyledButton} {
+      margin-left: 0px;
       &:first-child {
         margin-left: 0;
       }
     }
   `}
 `;
 
+export const StyledResponsiveFooterWrapper = styled.div`
+  margin: 0 auto;
```

Comment:
you are using margin and margin-top, maybe just use `margin: 20px auto 0 auto`

Reply:
Good point will give it a go thanks!